main.py

```diff
@@ -122,17 +122,27 @@ def cal_dividend_increament(div_obj: pd.core.series.Series, number_of_years: int
                    'is_monotonic': is_monotonic,
                    'is_persistent': is_persistent}
     return output_dict
-# %%
 
```

Comment:
the best thing to do here is to put all the functionality for importing the run_settings in a separate file and a separate class
you can call it pre_run.py